package.json

```diff
@@ -12,15 +12,24 @@
   "bugs": {
     "url": "https://github.com/RecordReplay/protocol/issues"
   },
+  "files": [
+    "js",
+    "json",
+    "LICENSE",
+    "README.md"
+  ],
   "main": "js/index.js",
   "scripts": {
     "build-json": "./build.sh",
     "build-ts": "ts-node build",
     "build-js": "tsc",
-    "build": "npm run build-json && npm run build-ts && npm run build-js"
+    "build": "npm run build-json && npm run build-ts && npm run build-js",
+    "clean": "rimraf ts/protocol ts/client/client.ts ts/client/generic.ts ts/server/event.ts ts/server/message.ts js",
```

Comment:
could this be `rimraf -r ts/**`?

Reply:
I moved the generated files into separate folders so that we don't have to list individual files anymore.